includes/class-wc-calypso-bridge-free-trial-store-details-task.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+
+/**
+ * Class WC_Calypso_Bridge_Free_Trial_Store_Details_Task.
+ *
+ * @since   2.0.12
+ * @version 2.0.12
```

Comment:
Since we're already at version 2.0.12, we should aim for the next version
```suggestion
 * @since   2.0.13
 * @version 2.0.13
```